ui/pages/login/redirect.js

```diff
@@ -13,14 +13,14 @@ export default function Redirect() {
   const [error, setError] = useState('')
 
   useEffect(() => {
+    // if it takes over 3 seconds to get the redirect values, something went wrong
```

Comment:
not related to this change, I just noticed this error flashes sometimes on redirect due to getting the cookie being async. Moved to a timed hook instead.

Reply:
Why is getting the cookie async? It should be immediately available in the browser unless it's populated by a request (doesn't seem like it here?)

Reply:
Got it, it was because the page re-rendered after I removed the cookie. Removed this timeout.